tests/MongoDB.Driver.Examples/CausalConsistencyExamples.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using Xunit;
+
+namespace MongoDB.Driver.Examples
+{
+    public class CausalConsistencyExamples
+    {
+        [Fact]
+        public void Causal_Consistency_Example_1()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            var client = GetClient();
+
+            string itemsCollectionName = "items";
+            var testDatabase = client.GetDatabase("test");
+            testDatabase.DropCollection(itemsCollectionName);
+
+            // Start Causal Consistency Example 1
+            using (var session1 = client.StartSession(new ClientSessionOptions { CausalConsistency = true }))
+            {
+                var currenDate = DateTime.UtcNow.Date;
```

Comment:
Fix typo, name should be "currentDate" (with a "t")

Reply:
:(